### DISCUSSION TITLE
I get an error when adding '.wezterm.lua' to my Home directory on macOS Ventura.

### DISCUSSION THREAD

**Discussion by conree** [wezterm/wezterm#5326] [General]
You can’t use a name that begins with a dot “.”, because these names are reserved for the system. Please choose another name.

**Comment by nekowinston** [comment]
[This is intentional](https://en.wikipedia.org/wiki/Hidden_file_and_hidden_directory#Unix_and_Unix-like_environments). Usually other applications on your Mac already use `~/.config/`, so you probably want to use `~/.config/wezterm/wezterm.lua` to avoid cluttering the $HOME directory.

Also see [Configuration Files](https://wezfurlong.org/wezterm/config/files.html#configuration-files) in the wezterm docs.
